new_lineage_panel/src/graph.ts

```diff
@@ -362,6 +364,39 @@ const processColumnLineage = async (
 
   const seeMoreLineage: ColumnLineage[] = [];
 
+  // since many edges can come to same node, one node can have multiple direct/indirect edges
+  // 1st pass to collect all type of edges that column at current level can have
+  const columnEdgeTypeCandidates: Record<string, string[]> = {};
```

Comment:
are there any e2e tests for checking these cases?

Reply:
currently no.